README.md

```diff
@@ -52,7 +52,8 @@ The `uno` command uses `conf/env.sh.example` for its default configuration which
 sufficient for most users.
 
 Optionally, you can customize this configuration by creating an `env.sh` file and modifying it for
-your environment:
+your environment. Inside this script the variable `UNO_HOME` defaults to the location of the
+cloned Uno repository. You may override this by defining UNO_HOME yourself, if need be.
```

Comment:
I agree, should not introduce without a good use case.  Adding it has maintenance implications.  In Accumulo @ctubbsii has discovered that ACCUMULO_HOME is not well defined and problematic for downstream packaging in Fedora.  For something like allowing Uno config to be stored in other locations, there are more targeted solutions that make less assumptions than `UNO_HOME`.